src/app/modules/bounce/bounce.controller.ts

```diff
@@ -23,11 +23,24 @@ export const handleSns: RequestHandler<
   // so we never fail on malformed input. The response code is meaningless since
   // it is meant to go back to AWS.
   try {
-    const isValid = await snsService.isValidSnsRequest(req.body)
-    if (!isValid) {
-      return res.sendStatus(StatusCodes.FORBIDDEN)
+    const isValid = await BounceService.isValidSnsRequest(req.body)
+    if (!isValid) return res.sendStatus(StatusCodes.FORBIDDEN)
+
+    const notification: IEmailNotification = JSON.parse(req.body.Message)
+    BounceService.logEmailNotification(notification)
+    const bounceDoc = await BounceService.getUpdatedBounceDoc(notification)
+    // Could not parse notification correctly
+    if (!bounceDoc) return res.sendStatus(StatusCodes.BAD_REQUEST)
+    // TODO (private #30): enable form deactivation
+    // if (bounceDoc.areAllPermanentBounces()) {
+    //   await BounceService.deactivateFormFromBounce(bounceDoc)
```

Comment:
Don't leave commented out code. Just go ahead and delete unused code. Now that we are using a revision control system, we can recover any deleted code later.

Reply:
uncommented, since we decided to go ahead an implement for this PR